src/LearnositySdk/Request/Init.php

```diff
@@ -140,6 +143,49 @@ public function __construct($service, $securityPacket, $secret, $requestPacket =
         $this->securityPacket['signature'] = $this->generateSignature();
     }
 
+    /**
+     * Adds metadata to request packet to enable telemetry and tracking. Request packet will be
+     * extended with following parameters:
+     * {
+     *   "meta": {
+     *     "sdk": {
+     *       "version": "v0.10.0",
+     *       "lang": "php",
+     *       "lang_version": "5.6.36",
+     *       "platform": "Linux",
+     *       "platform_version": "#1 SMP Tue Jun 26 16:32:21 UTC 2018"
+     *     }
+     *   }
+     * }
+     *
+     * @param array    $requestPacket
+     *
+     * @return array
+     */
+    private function addMeta(array $requestPacket) {
+        $requestPacket['meta'] = [
+            'sdk' => [
+                'version' => $this->getSDKVersion(),
+                'lang' => 'php',
+                'lang_version' => phpversion(),
+                'platform' => php_uname('s'),
+                'platform_version' => php_uname('r')
+            ]
+        ];
+
+        return $requestPacket;
+    }
+
+    private function getSDKVersion() {
+        $path = dirname(__FILE__) . '/../../../.version';
```

Comment:
@shtrom dunno if this is a best approach to construct paths in PHP, so happy to hear some feedback :)

Reply:
👍